.github/workflows/pipelines.yml

```
@@ -344,16 +348,43 @@ jobs:
+          GH_ARTIFACT_TOKEN: ${{ fromJson(steps.pipelines-tokens.outputs.tokens_json).customer_org_read }}
+          PIPELINES_PLAN_ENCRYPTION_KEY: ${{ secrets.PIPELINES_PLAN_ENCRYPTION_KEY }}
+
       - name: Check Status
+        if: always()
         shell: bash
         working-directory: ./infra-live-repo
         env:
           API_BASE_URL: ${{ inputs.api_base_url }}
           GH_TOKEN: ${{ fromJson(steps.pipelines-tokens.outputs.tokens_json).propose_infra_change }}
           GH_ARTIFACT_TOKEN: ${{ fromJson(steps.pipelines-tokens.outputs.tokens_json).propose_infra_change }}
+          AFTER_HOOKS_OUTPUT_FILE: ${{ steps.hooks_after.output_file }}
```

actionlint (1.7.12)

[error] property "output_file" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)